share/instruments/instruments.xml

```diff
@@ -7693,10 +7693,10 @@
             </Instrument>
             <Instrument id="crash-cymbal">
                   <family>unpitched-metal-percussion</family>
-                  <trackName>Crash Cymbal</trackName>
-                  <longName>Crash Cymbal</longName>
-                  <shortName>Cr. Cym.</shortName>
-                  <description>Crash cymbal.</description>
+                  <trackName>Suspended Cymbal</trackName>
+                  <longName>Suspended Cymbal</longName>
+                  <shortName>Sus. Cym.</shortName>
+                  <description>Suspended Cymbal</description>
```

Comment:
Missing full stop (period) at end of description.

Some descriptions include multiple sentences, so we decided that all descriptions should end with a full stop, even when there's only one sentence.